scripts/boilerplate_generator.py

```diff
@@ -196,12 +196,12 @@ def get_funcs_info(fn_infos, module_name):
         ret += '    "{0.name}",\n'.format(info)
     ret += "    NULL};\n\n"
 
-    ret += ("const gchar* const* get_{0}_functions (void) {{\n".format(module_name) +
+    ret += ("static const gchar* const* get_{0}_functions (void) {{\n".format(module_name) +
             "    return {0}_functions;\n".format(module_name) +
             "}\n\n")
 
     ret += "static const guint8 {0}_num_functions = {1};\n\n".format(module_name, len(fn_infos))
-    ret += ("guint8 get_{0}_num_functions (void) {{\n".format(module_name) +
+    ret += ("static guint8 get_{0}_num_functions (void) {{\n".format(module_name) +
             "    return {0}_num_functions;\n".format(module_name) +
             "}\n\n")
```

Comment:
These two should probably be removed, we are not using them anywhere, AFAICT.

Reply:
Ok, removed.